src/vs/workbench/browser/parts/titlebar/titlebarPart.ts

```diff
@@ -713,19 +710,19 @@ export class BrowserTitlebarPart extends Part implements ITitlebarPart {
 		return !this.isAuxiliary && this.configurationService.getValue(LayoutSettings.ACTIVITY_BAR_LOCATION) === ActivityBarPosition.TOP;
 	}
 
-	protected get useCounterZoom(): boolean {
+	get hasNoElementsToZoom(): boolean {
+		const noMenubar = this.currentMenubarVisibility === 'hidden' || this.currentMenubarVisibility === 'compact' || (!isWeb && isMacintosh);
+		const noCommandCenter = !this.isCommandCenterVisible;
+		const noToolBarActions = !this.layoutControlEnabled && !this.editorActionsEnabled && !this.activityActionsEnabled;
+		return (noMenubar && noCommandCenter && noToolBarActions);
+	}
 
+	get useCounterZoom(): boolean {
```

Comment:
Nit: I never fully got what "counter zoom" actually means. Can we find a better name?